### ISSUE TITLE
Install with opam package

### ISSUE THREAD

**Issue by clarus** [rems-project/lem#24]
Is there an opam package for `lem` published on the main repository?

What I have seen:
* an `opam` file at the root of the project;
* a comment about a planned package in https://github.com/rems-project/lem/pull/9 ;
* a package [sibylfs-lem](https://opam.ocaml.org/packages/sibylfs-lem/) which I believe is not the official one: it says "SibylFS fork".

**Comment by rmn30** [comment]
The opam package is released in our private [opam repository](https://github.com/rems-project/opam-repository) which you can use with `opam repository add`.

**Comment by clarus** [comment]
OK, I have not seen it. Some things I am wondering:
* could this repository be a fresh repository with only the `rems-project` package, rather than a fork? For example, as of today, the changes in the fork conflict with the mainstream changes and I do not know how opam handles that;
* could the `rems-project` packages be published on the main opam repository? Even if the repository stays separated, I was thinking about prefixing all the packages by `rems-project-` to namespace them;
* should there be some opam install instructions in the README?

I can make pull-requests.

**Comment by rmn30** [comment]
We are new(ish) to opam packaging so are just learning!

> could this repository be a fresh repository with only the rems-project package, rather than a fork? For example, as of today, the changes in the fork conflict with the mainstream changes and I do not know how opam handles that;

Yes it could but forking seemed like the easiest way to replicate the repository layout -- I just deleted all the packages that aren't ours to speed up loading. We've found that opam can handle multiple repositories with no problems. Until we want to add our packages to the main opam repository I don't think there's a reason to pull from upstream.

> could the rems-project packages be published on the main opam repository? 

They could but we'd probably have to do some work before they were accepted, for example the repository format has changed for opam 2 and I'm not sure what we'd need to change to be compatible. Right now our repository works with opam 1.2 and opam 2 which is quite useful.

> Even if the repository stays separated, I was thinking about prefixing all the packages by rems-project- to namespace them;

We could do that but unless there are conflicts I prefer the short names.

> should there be some opam install instructions in the README?

Yes there probably should be. At the time of creating the packages we were focused on packaging [sail](/rems-project/sail) and its dependencies so didn't mention it here.
